src/App.tsx

```diff
@@ -15,61 +16,136 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-enum SortType {
+enum sortType {
```

Comment:
Let's name types starting with capital letter
```suggestion
enum SortType {
```